clients/include/testing_gemm_ex.hpp

```diff
@@ -132,18 +133,22 @@ inline hipblasStatus_t testing_gemm_ex_template(const Arguments& arg)
                                             M,
                                             N,
                                             K,
-                                            &h_alpha_Tc,
+                                            &h_alpha_Tex,
                                             dA,
                                             a_type,
                                             lda,
                                             dB,
                                             b_type,
                                             ldb,
-                                            &h_beta_Tc,
+                                            &h_beta_Tex,
                                             dC,
                                             c_type,
                                             ldc,
+#ifdef HIPBLAS_V2
```

Comment:
should a_type, b_type, c_type be hipDatatype when HIPBLAS_V2 is defined?

Reply:
Yes, but in hipblas.h I have:
```
#ifdef HIPBLAS_V2
typedef hipDataType hipblasDatatype_t;
...
#endif
```
To make the switch easier.